morpheus-graphql-core/src/Data/Morpheus/Types/Internal/AST/Error.hs

```diff
@@ -62,6 +63,10 @@ atPositions GQLError {..} pos = case toList pos of
   posList -> GQLError {locations = locations <> Just posList, ..}
 {-# INLINE atPositions #-}
 
+
+withExtensions :: GQLError -> Map Text Value -> GQLError
+withExtensions gqlerr ext = gqlerr { extensions = Just ext }
```

Comment:
we will overwrite all existing extension with it. is it behavior that you intended?
what about this?
```hs
withExtensions err ext = err { extensions = extensions err  <> Just ext }
```

Reply:
The behaviour is modelled after the `withPath` function. I'm happy with either solution

Reply:
would you like me to change it to your suggestion? :)

Reply:
yes. that would be right way to go. i think :)